sdk/python/tests/test_offline_online_store_consistency.py

```diff
@@ -196,7 +196,7 @@ def check_offline_and_online_features(
 ) -> None:
     # Check online store
     response_dict = fs.get_online_features(
-        [f"{fv.name}:value"], [{"driver": driver_id}]
+        [f"{fv.name}:value"], [{"driver": driver_id}], full_feature_names=True
```

Comment:
since this test isn't focused on testing the full_feature_names param, it should just use the default so the full_feature_names param doesn't need to be set in this test, and you can modify the expected feature names.